seed_isort_config.py

```diff
@@ -8,11 +8,16 @@
 import os.path
 import re
 import subprocess
+from collections import OrderedDict
 
 from aspy.refactor_imports.classify import classify_import
 from aspy.refactor_imports.classify import ImportType
 
-
+CONFIG_FILE_TO_START_RE = OrderedDict([
```

Comment:
what does this benefit from being an `OrderedDict`?

Reply:
and would a tuple-of-tuples suffice?

Reply:
The mapping makes the `conf_start = CONFIG_FILE_TO_START_RE[conf_filename]` lookup fast, and the ordering ensures that we iterate through `CONFIG_FILE_TO_START_RE` in a deterministic way. But I could use nested tuples